app/components/bookings/create/create-booking.component.ts

```diff
@@ -1,9 +1,130 @@
 import {Component} from '@angular/core';
+import {Workpackage} from "../../../models/workpackage/workpackage";
+import {NavParams, NavController, Alert} from "ionic-angular/index";
+import {BookingService} from "../../../services/bookings/booking.service";
+import {AbstractControl, ControlGroup, FormBuilder, Validators} from "@angular/common";
+import moment = require("moment/moment");
+import {BookingSessionService} from "../../../services/bookings/booking-session.service";
+import {BookingSession, Booking} from "../../../models/booking/booking";
+import Moment = moment.Moment;
+import {Response} from "@angular/http";
 
 @Component({
   templateUrl: 'build/components/bookings/create/create-booking.component.html'
 })
 export class CreateBookingComponent {
-  constructor() {
+  private _workpackage: Workpackage;
+  private _bookingSession: BookingSession;
+  effort : AbstractControl;
```

Comment:
these properties could be private. Also i think something like effortFormControl, would be an more appropriate name.